gradle/versions.gradle

```diff
@@ -54,7 +54,13 @@ dependencyManagement {
 
     dependency 'junit:junit:4.12'
 
-    dependency 'net.consensys.cava:cava-toml:0.5.0'
+    dependencySet(group: 'org.apache.tuweni', version: '0.9.0-20190709195335') {
```

Comment:
Is this a full release or a snapshot?  The -201907... bit has me worried.

Reply:
This to me indicates a non-production release.  And being an apache project it may not have had its release approved.  This makes me nervous.

Is there a particular reason we need to go to the tuweni library?

Reply:
The Cava project is archived, I think this forces us to go to where the project lives now. I would prefer an incubating version over an archived one.
Binaries are all named after this scheme : https://bintray.com/tuweni/tuweni/tuweni
Also Besu is in the same incubation status at Hyperledger, it doesn't make it a project people can't use, does it?

Reply:
Incubation isn't the issue, the question is whether this is a stable release or a -SNAPSHOT.  We'd much prefer a stable release of an archived project to a -SNAPSHOT release.

Reply:
Notably, the latest release on Tuweni's GitHub is 0.8.1: https://github.com/apache/incubator-tuweni/releases